tensorly/backend/pytorch_backend.py

```diff
@@ -174,133 +184,22 @@ def argmax(input, axis=None):
     def stack(arrays, axis=0):
         return torch.stack(arrays, dim=axis)
 
-    @staticmethod
-    def _reverse(tensor, axis=0):
-        """Reverses the elements along the specified dimension
-
-        Parameters
-        ----------
-        tensor : tl.tensor
-        axis : int, default is 0
-            axis along which to reverse the ordering of the elements
-
-        Returns
-        -------
-        reversed_tensor : for a 1-D tensor, returns the equivalent of
-                        tensor[::-1] in NumPy
-        """
-        indices = torch.arange(tensor.shape[axis] - 1, -1, -1, dtype=torch.int64)
-        return tensor.index_select(axis, indices)
+    def svd(self, X, full_matrices=True):
+        # The torch SVD has accuracy issues. Try again when torch.linalg is stable.
+        ctx = self.context(X)
+        X = self.to_numpy(X)
```

Comment:
Pytorch 1.8 is now the stable release, should we consider replacing this with `torch.linalg.svd`? 
As a side note, the linear algebra functions in torch will be deprecated with their `linalg` counterparts in 1.9. For example, `torch.qr` is already deprecated in 1.8 we should replace it with its `linalg` equivalent.

Reply:
Yes, I was actually looking forward to switching `torch.linalg.svd`! Thanks @merajhashemi !

Reply:
Thanks for the note @merajhashemi. I'll open an issue to make sure this is done.